hooks/time.js

```diff
@@ -0,0 +1,3 @@
+export const useformatDate = (time) => {
```

Comment:
You can make it more flexible by receiving the formatting option, too. And maybe a better name:
```javascript
export const useDateFormatter = (date, options = {}) => {
  return new Intl.DateTimeFormat().format(new Date(date), options)
}
```